tests/test_data/test_datasets/test_sot_dataset.py

```diff
@@ -10,105 +10,124 @@
 
 PREFIX = osp.join(osp.dirname(__file__), '../../data')
 SOT_DATA_PREFIX = f'{PREFIX}/demo_sot_data'
-DATASET_NAME_MAP = dict(GOT10kDataset='got10k')
+DATASET_INFOS = dict(
+    GOT10kDataset=dict(img_prefix=osp.join(SOT_DATA_PREFIX, 'got10k')),
+    TrackingNetDataset=dict(
+        num_chunks=1, img_prefix=osp.join(SOT_DATA_PREFIX, 'trackingnet')),
+    SOTCocoDataset=dict(
+        ann_file=osp.join(SOT_DATA_PREFIX, 'lasot', 'lasot_test_dummy.json'),
+        img_prefix=osp.join(SOT_DATA_PREFIX, 'lasot')),
+    SOTImageNetVIDDataset=dict(
+        ann_file=osp.join(SOT_DATA_PREFIX, 'lasot', 'lasot_test_dummy.json'),
+        img_prefix=osp.join(SOT_DATA_PREFIX, 'lasot')))
```

Comment:
use the coco_vid.json in [here](https://github.com/open-mmlab/mmtracking/blob/master/tests/data/demo_cocovid_data/ann.json)